.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+    "presets": ["env"] 
```

Comment:
Check this out... https://babeljs.io/en/setup#installation
![Uploading Screenshot 2020-12-03 at 08.44.18.png…]()
 I don't this is the right present for @babel/preset-env